src/entity-manager/EntityManager.ts

```diff
@@ -178,7 +180,9 @@ export class EntityManager extends BaseEntityManager {
         } else {
             // todo: throw exception if constructor in target is not set
             if (target instanceof Array) {
-                return this.getRepository<Entity[]>(target[0].constructor).remove(entity as Entity[], options);
+                return Promise.all(target.map((t, i) => {
```

Comment:
its really not something important. But if you want to lessen my load there are bunch of issues needs to be fixed from easy to hard. Just let me know or pick any 😃